routes/auth.js

```diff
@@ -54,10 +54,10 @@ router.get('/github/start', passportOrPat(), (req, res) => {
 
 router.get('/github/finalize', passportOrPat(), async (req, res) => {
   const token = req.user.githubAccessToken
-  const permissions = await getPermissions(token, options.org)
+  const { publicEmails, permissions } = await getPermissions(token, options.org)
```

Comment:
getPermissions should either be renamed or split into two functions since it now gets both permissions and public emails.

Reply:
I've done it order to avoid 2 separate calls to the APIs , we can think on renaming it in getUserDetails perhaps?